ci/yocto-check-layer.sh

```diff
@@ -4,17 +4,22 @@
 
 TOPDIR=$(realpath $(dirname $(readlink -f $0))/..)
 
+# Ensure KAS workspace is outside of the checked out repo
+# Allows the caller to specify KAS_WORK_DIR, otherwise make temp one
+export KAS_WORK_DIR=$(realpath ${KAS_WORK_DIR:-$(mktemp -d -p $TOPDIR/../)})
```

Comment:
I think we should prefer the $PWD when KAS_WORK_DIR is not set, instead of creating a new folder outside the repo. If the local user want to run this somewhere other than the default KAS_WORK_DIR, he should be the one to choose this option.

Reply:
but PWD is the path to meta-qcom-hwe, no? We need the kas workspace to be outside of the repo, so that poky and meta-qcom are not checked out inside meta-qcom-hwe. Maybe I did not understand what you meant here.

Reply:
another option would be to leave that out of the ci scripts, and set KAS_WORK_DIR in the github actions. e.g. leave that up to the user the configure

Reply:
I think this is fine, I would just prefer the temp directory to be created in /tmp/ instead.

Reply:
but if it runs with PWD by default (assuming from meta-qcom-hwe) it can end up failing and confuse the user.

Reply:
yes, you are right and this can happen. we can have different results between locally and ci.
so in the end /tmp is more similar to what we have on ci